src/transaction/buffer.rs

```diff
@@ -275,6 +275,20 @@ impl Buffer {
             }
         }
     }
+
+    pub async fn get_size(&self) -> u64 {
```

Comment:
@ekexium Is it closer to convention?

```suggestion
    pub async fn get_write_size(&self) -> u64 {
```

Reply:
I prefer `get_write_size`. It's more accurate

Reply:
I have changed it to `get_write_size`